nc_time_axis/__init__.py

```diff
@@ -15,6 +15,11 @@
 import cftime
 import numpy as np
 
+try:
+    from cftime import datetime_base as cftime_datetime
+except ImportError:
+    from cftime import datetime as cftime_datetime
+
```

Comment:
This would be better sanitised by actually checking the `cftime` version to determine the correct behaviour (using [packaging.version.parse](https://packaging.pypa.io/en/latest/version/#packaging.version.parse) or similar).